pkg/chaosctl/cmd/logs.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
 	"github.com/chaos-mesh/chaos-mesh/controllers/config"
 	cm "github.com/chaos-mesh/chaos-mesh/pkg/chaosctl/common"
-	"github.com/chaos-mesh/chaos-mesh/pkg/utils"
+	sele "github.com/chaos-mesh/chaos-mesh/pkg/selector"
```

Comment:
This abbr sounds a little weird. 🙈 How about using the full "selector".

Reply:
Fixed